### PR TITLE
Fix OSS integration tests for docs (#3663)

### PR DESCRIPTION
**Pull request by tkorenko**
## What
+ In "Docs PR check mode" stop before "yarn deploy"

### CODE REVIEW

**Review by supertopher — APPROVED** [review]
taras can we make this run locally without a failure?  

otherwise looks great

File: tools/bin/deploy_docusaurus [hunk 1]
```diff
@@ -79,6 +79,12 @@ if ! yarn build; then
   exit 1
 fi
 
+# "check PR mode" -- we run "yarn build" and stop before "yarn deploy"
+if [ "${SKIP_DEPLOY}" = "yes" ]; then
```
Comment (supertopher) [inline tools/bin/deploy_docusaurus:83]: SKIP_DEPLOY doesn't exist for local runs

Comment (tkorenko) [reply tools/bin/deploy_docusaurus:83]: which is perfectly fine for local deploys

Comment (tkorenko) [reply tools/bin/deploy_docusaurus:83]: we get then `if [ "" = "yes" ]; then ... "

**Pull request merged**